tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -114,12 +115,14 @@ Tensor log1p(const Tensor& x, const MemoryConfig& output_mem_config) {
 // use transformation y = log[1+exp[x]] by broadcast
 Tensor _softplus(const Tensor& x, float beta, float threshold, const MemoryConfig& output_mem_config) {
     float oned_beta = (1 / beta);
-    Tensor x_beta = mul_unary(x, beta, output_mem_config);
+    auto x_4D = ttnn::unsqueeze_to_4D(x);
```

Comment:
If possible, I'd avoid adding ttnn operations into metal.  Could we do the unsqueeze and squeeze one level higher? What do you think @arakhmati @cfjchu ?

Reply:
@arakhmati clarified that this would block the main thread for those using this through tt_lib api and recommended to move into the ttnn. Doing this now. 
Thank you!

Reply:
@xanderchin, done!